pkg/routing/routing_test.go

```diff
@@ -50,14 +51,19 @@ func TestGetRouteConsistency(t *testing.T) {
 	table, err := GetRoutingTable(ctx)
 	assert.NoError(t, err)
 	for _, route := range table {
-		if ip := route.LocalIP.To4(); ip != nil {
+		if ip := route.RoutedNet.IP.To4(); ip != nil {
+			if ip.String() == "0.0.0.0" || ip.IsMulticast() {
+				// Don't test 0.0.0.0 or any multicast addresses.
+				continue
+			}
```

Comment:
this is the fix. It never made any sense to test the LocalIP, what we actually care about is the subnet that's being routed. As for 0.0.0.0 and Multicast, windows fails to route those when given as addresses.